src/main/java/gndata/lib/util/OntologyHelper.java

```diff
@@ -0,0 +1,103 @@
+package gndata.lib.util;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import com.hp.hpl.jena.ontology.*;
+import com.hp.hpl.jena.rdf.model.Resource;
+import com.hp.hpl.jena.vocabulary.RDF;
+
+/**
+ * Created by andrey on 11.05.15.
+ */
+public class OntologyHelper {
+
+    private OntModel ontology;
+
+    public OntologyHelper(OntModel ontology) {
+        this.ontology = ontology;
+    }
+
+    /* ontology Class methods */
+
+    public List<OntClass> listClasses() {
+        return ontology.listClasses().toList();
```

Comment:
Yields only results at all when I add
OntModel schema = ModelFactory.createOntologyModel(OntModelSpec.OWL_MEM_RDFS_INF);
to the MetadataService